code/game/objects/random/biomes/guns.dm

```diff
@@ -1,6 +1,6 @@
 /obj/landmark/loot_biomes/obj/guns
 	name = "Biome: Guns"
-	main_tags = list(SPAWN_GUN = 2, SPAWN_GUN_ENERGY, SPAWN_GUN_SHOTGUN)
+	main_tags = list(SPAWN_GUN_HANDMADE)
```

Comment:
Correct me if I'm wrong, but aren't biomes mainly used in deepmaint to determine the amount of traps and mobs?
To quote evankhell,

> biome spawners: These are a smart spawner that prioritizes the most dangerous mobs and traps depending on the amount of loot in the area.

Reply:
I think it affects everything, both Eris and deepmaint and stuff. Eh, if anything, I'll make a hotfix; bringing back the old main tags to this file.

Reply:
preety sure they're use for junk beacon

Reply:
> preety sure they're use for junk beacon

It has children to spawn roaches and spiders, which alongside Evankhell's comment is why I assume this is used in deepmaint.